server/serverMethods.js

```diff
@@ -755,6 +755,9 @@ function ServerMethods(aLogLevel, aModules) {
   function postRoomArchive(aReq, aRes) {
     const { tbConfig } = aReq;
     const { body } = aReq;
+
+    if (!aReq.tbConfig.enableArchiving) return aRes.status(403).send(new Error('Unauthorized'));
```

Comment:
I see this test in server_spec.js:

```
  it('POST /room/:roomName/archive should allow individual archiving', (done) => {
    request(app)
      .post('/room/unitTestRoom/archive')
      .send('userName=xxxYYY&operation=startIndividual')
      .expect(checkForAttributes.bind(undefined, ArchiveInfo))
      .expect(200, done);
  });
```
Can we add another test for this response when `enableArchiving` is not `true`?